apps/full-site-editing/full-site-editing-plugin/premium-content/blocks/logged-out-view/submit-buttons.js

```diff
@@ -0,0 +1,188 @@
+ */
+function SubmitButtons( props ) {
+	useEffect( () => {
+		setButtonClasses();
+		setCustomButtonColors();
+	}, [ props.backgroundButtonColor, props.textButtonColor ] );
+
+	function setButtonClasses() {
+		const buttonClasses = getButtonClasses();
+		props.setAttributes( { buttonClasses } );
+	}
+
+	function setCustomButtonColors() {
+		const customTextButtonColor = getTextButtonColor();
+		const customBackgroundButtonColor = getBackgroundButtonColor();
+
+		if ( customTextButtonColor !== undefined ) {
+			props.setAttributes( { customTextButtonColor } );
+		}
+
+		if ( customBackgroundButtonColor !== undefined ) {
+			props.setAttributes( { customBackgroundButtonColor } );
+		}
+	}
+
+	function getTextButtonColor() {
+		return get( props.textButtonColor, 'color' );
+	}
+
+	function getBackgroundButtonColor() {
+		return get( props.backgroundButtonColor, 'color' );
+	}
+
+	function getButtonClasses() {
+		const { textButtonColor, backgroundButtonColor } = props;
+		const textClass = get( textButtonColor, 'class' );
+		const backgroundClass = get( backgroundButtonColor, 'class' );
+		return classnames( 'wp-block-button__link', {
+			'has-text-color': textButtonColor.color,
+			[ textClass ]: textClass,
+			'has-background': backgroundButtonColor.color,
+			[ backgroundClass ]: backgroundClass,
+		} );
+	}
+
+	const {
+		attributes,
+		setAttributes,
+		backgroundButtonColor,
+		textButtonColor,
+		setBackgroundButtonColor,
+		setTextButtonColor,
+		fallbackBackgroundColor,
+		fallbackTextColor,
+	} = props;
+
+	const backgroundColor = backgroundButtonColor.color || fallbackBackgroundColor;
+	const color = textButtonColor.color || fallbackTextColor;
+	const buttonStyle = { border: 'none', backgroundColor, color };
+	const buttonClasses = getButtonClasses();
+
+	return (
+		<div>
+			<div className="wp-block-button premium-content-logged-out-view-button">
+				<RichText
+					placeholder={ __( 'Add text…', 'full-site-editing' ) }
+					value={ attributes.subscribeButtonText }
+					onChange={ ( nextValue ) => setAttributes( { subscribeButtonText: nextValue } ) }
+					className={ buttonClasses }
+					style={ buttonStyle }
+					keepPlaceholderOnFocus
+				/>
+				<RichText
+					placeholder={ __( 'Add text…', 'full-site-editing' ) }
+					value={ attributes.loginButtonText }
+					onChange={ ( nextValue ) => setAttributes( { loginButtonText: nextValue } ) }
+					className={ buttonClasses }
+					style={ buttonStyle }
+					keepPlaceholderOnFocus
+				/>
+			</div>
+			<InspectorControls>
+				<PanelColorSettings
+					title={ __( 'Button Color Settings', 'full-site-editing' ) }
+					colorSettings={ [
+						{
+							value: backgroundButtonColor || undefined,
+							onChange: setBackgroundButtonColor,
+							label: __( 'Background Color', 'full-site-editing' ),
+						},
+						{
+							value: textButtonColor || undefined,
+							onChange: setTextButtonColor,
+							label: __( 'Text Color', 'full-site-editing' ),
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **130** times:
`translate( 'Text Color' )` *ES Score: 10*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/fix/premium-content-fallback)